C-array-every/exercise.js

```diff
@@ -5,7 +5,11 @@
 var students = ["Omar", "Austine", "Dany", "Swathi", "Lesley", "Rukmini"];
 var group = ["Austine", "Dany", "Swathi", "Daniel"];
 
-var groupIsOnlyStudents; // complete this statement
+var groupIsOnlyStudents = group.every(function(students) {
```

Comment:
Not quite there. What should the function passed to every do here? What are its input (parameter) and output (return value)?